Assets/Scripts/Game.cs

```diff
@@ -72,6 +75,13 @@ void Start()
         // Create some things
         uiScaler = new UIScaler(uICanvas);
         config = new ConfigFile();
+        currentLang = 1; // English
+
+        if (config.data.Get("MoMConfig") != null)
```

Comment:
I will remove the extract-ini.files , shouldn't it be in the .gitignore file in order to not be included in git branches?